src/components/AlumniTimeline.tsx

```diff
@@ -1,18 +1,24 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import styles from './AlumniTimeline.module.css';
-import Members from "./Members";
+import MemberList from "./Members";
 
-const AlumniTimeline = ({ alumni }) => {
+import type { alumniData, alumniOfYear } from "../data/alumniData";
+
+type AlumniProps = {
+  readonly alumni: alumniData;
+};
+
+const AlumniTimeline = ({ alumni }: AlumniProps): ReactElement => {
   return (
     <div className={styles.container}>
       <ul className={styles.timeline}>
         {
-          alumni.map(({ classOf, members }) => (
+          alumni.map(({ classOf, members }: alumniOfYear): ReactElement => (
```

Comment:
These type annotations are not really necessary. TypeScript can use type inference to figure out from the context

Reply:
Thanks for pointing that out, will remove them in the next commit